metadetect/tests/test_lsst_metadetect.py

```diff
@@ -143,8 +143,8 @@ def test_lsst_metadetect_mfrac_ormask(cls):
         for shear in ["noshear", "1p", "1m", "2p", "2m"]:
             assert np.any(res[shear]["flags"] == 0)
             assert np.all(
-                (res[shear]["mfrac"] > 0.45)
-                & (res[shear]["mfrac"] < 0.55)
+                (res[shear]["mfrac"] > 0.40)
+                & (res[shear]["mfrac"] < 0.60)
```

Comment:
This change in mfrac seems unrelated to this PR. What happened here?

Reply:
I interpreted it as related to some noise difference.   The same seed may have resulted in slightly different measurements due to small changes in code order

Reply:
What do you think, is that convincing?

Reply:
I noticed that failed in your PR as well, so may be related to the stack version

Reply:
It just seems rather odd. :/

Reply:
I agree.  Since it affects different unrelated PRs it seems unrelated to this repo at all

Reply:
Let's not delay and just go for it. If it changes again we can dig.